requirements.lock

```diff
@@ -1,27 +1,27 @@
-certifi==2019.11.28
```

Comment:
Can you undo all these lockfile changes until we can sort out a better approach to this?  Major version updates should, at the least, include some slack discussion and their own PR